node-intro/hello-node.js

```diff
@@ -0,0 +1 @@
+console.log();
```

Comment:
Don't forget to modify `hello-node.js`:

---

![image](https://github.com/claylevy53/m2-0124-code-solutions/assets/16143771/7e56c986-95f8-479f-b017-80124e04e11d)